pkgs/desktops/gnome/core/evolution-data-server/default.nix

```diff
@@ -38,7 +38,7 @@ stdenv.mkDerivation rec {
     libcanberra-gtk3 pcre libphonenumber boost protobuf
   ];
 
-  propagatedBuildInputs = [ libsecret nss nspr libical db libsoup ];
+  propagatedBuildInputs = [ libsecret nss nspr libical db libsoup libgdata ];
```

Comment:
Could you please move it onto a separate line and add the explanation into the source code? It will be more easily visible than trying `git blame` on a single line that somewhat often changes.

Reply:
Done.